SEP-0002-revised.md

```diff
@@ -0,0 +1,253 @@
+# SEP-0002 - SunPy Organization Definition
+
+# Abstract
+The SunPy project has changed significantly since the first version of SEP-0002 was written in 2014.
+This SEP details a new governance structure designed to enable the project to grow, while allowing for more input from the wider SunPy community.
+
+## Detailed Description
+The primary role of the SunPy Project is to *facilitate and promote the use and development of community-led, free and open-source solar data-analysis software based on the scientific Python environment*.
+This includes but is not limited to the following tasks:
+
+* developing the sunpy core library and other affiliated packages
+* manage and protect the SunPy brand and identity
+* promote SunPy to the community
+* managing SunPy assets, i.e. GitHub organization, domain name etc.
+
+# Governance Structure
+
+The SunPy project's governance involves the following five components:
+
+* The SunPy Community
+* The Project Members
+* The Steering Committee
+* The Advisory Board
+* The Ombudsperson
+
+The SunPy project exists for the benefit of the whole community: users, developers, and members of the wider solar physics and scientific software community.
+This community is the ultimate authority in how the project is run, this authority is exercised through the processes outlined in this document, primarily to the Steering Committee and the Project Members.
+Whenever possible all decisions taken should be done by consensus, considering input from both active contributors and the wider community, rather than taken in isolation by the projects governance.
+
+## Advisory Board
+The primary functions of the advisory board are: to advise the steering committee on management of the project, find funding for the project to ensure the core aims of the project can be met, nominate a Ombudsperson, and to ensure that the governance of the project is running as intended.
+The board is self-electing, with members appointed by a vote of the existing board.
+
+* **Membership Limit**: The board shall consist of at least 3 members and no more than 7 members.
```

Comment:
I realize that you're just carrying forward the language in the existing SEP-0002, but I'm confused by the some of the later language given that the board can have variable size.  My questions:

- If the board currently has 4 members, and thus 3 open slots, are those open slots "vacancies" or are they "previously unfilled board positions", which are apparently distinct concepts?  The board is supposed to fill vacancies "as soon as possible", but what about "previously unfilled board positions"?
- If the board currently has 4 members, and one member's two-year term has expired and is extended pending a successor, how is "successor" defined?  Is any new member automatically a successor (e.g., even if there is no overlap in the cross-section of the community that they represent)?  Can a new member be appointed to a "previously unfilled board position" rather than to the "vacancy" that would be left by the expired member's leaving, and thus not actually trigger the end of the expired member's extended term?
- If the board currently has 4 members, and one person leaves, is that a vacancy that should be filled as soon as possible?  Or can the board simply say that it is now happy with 3 members, so there actually isn't any vacancy?

Reply:
This section has now been rewritten by @ehsteve and me. I think we have clarified all these points, can you have a read?

Reply:
Yup, better